src/django/api/migrations/0158_create_moderation_events_table.py

```diff
@@ -0,0 +1,34 @@
+# Generated by Django 3.2.17 on 2024-10-14 10:04
+
+import django.contrib.postgres.fields
+import django.contrib.postgres.indexes
+from django.db import migrations, models
+import django.db.models.deletion
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('api', '0157_delete_endpoint_switcher_for_list_uploads'),
+    ]
+
+    operations = [
+        migrations.CreateModel(
+            name='ModerationEvent',
+            fields=[
```

Comment:
1. Added `request_type` instead of just `type`.
2. Added `cleaned_data` field.
3. Added `geocode_result` field.
4.  Added `uuid` field using uuid Python package.